build/cqs.tsv

```diff
@@ -47,7 +46,7 @@ com.sun.xml.bind:jaxb-impl	2.2.11	compile
 com.thoughtworks.paranamer:paranamer	2.6	compile
 com.typesafe:config	1.2.1	compile
 com.typesafe.scala-logging:scala-logging_2.11	3.1.0	compile
-com.vividsolutions:jts	1.13	compile
+com.vividsolutions:jts-core	1.14.0	compile
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14205

Reply:
this didn't pass the automated scanning...

Reply:
Has been certified! (https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14205 has been updated)

Reply:
nice, that was pretty fast